src/style.css

```diff
@@ -0,0 +1,87 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  color: #fff;
+  font-size: 100px;
+}
+
+.blocks {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.block {
+  height: 300px;
+  min-width: 300px;
+  flex-basis: 33.33%;
+  flex-grow: 1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.block:nth-child(1) {
+  background-color: rgb(100%, 0%, 0%);
+}
+
+.block:nth-child(2) {
+  background-color: rgb(80%, 0%, 0%);
+}
+
+.block:nth-child(3) {
+  background-color: rgb(60%, 0%, 0%);
+}
+
+.block:nth-child(4) {
+  background-color: rgb(40%, 0%, 0%);
+}
+
+.block:nth-child(5) {
+  background-color: rgb(20%, 0%, 0%);
+}
+
+.block:nth-child(6) {
+  background-color: #000;
+}
+
+@media (max-width: 599px) {
+  .block {
+    flex-basis: 100%;
+  }
+}
```

Comment:
If you want to use a mobile-first approach, style your blocks with 100% width by default. Don't add too many media queries